### ISSUE TITLE
Feature Request: Rearranging Account Folders

### ISSUE THREAD

**Issue by JustAnonymous** [k9mail/k-9#3242]
I have an account with a number of aliases, each has a separate inbox and sent folder.  The folders are shown all mixed together in K9, and not in the clean order they are on the server and in thunderbird. In K9 to change this I only have the option of "Show in top group".  If I do that to all of the inboxes, the other sent folders are mixed in with drafts, trash etc.  Would be nice if the Folders could be individually rearranged manually.  I have 12 folders, could imagine it would start to get really ugly with more.

### Expected behavior
Users should have the option to rearrange folders manually.  Or at least have them arranged as they are on the server.  I'm guessing they're currently alphabetical? Sort of? 

### Actual behavior
Selecting "Show in top group" is the only option, and this doesn't allow clean organization of multiple folders under a single account.

### Steps to reproduce
1.
2.
3.

### Environment
K-9 Mail version: 5.501

Android version: 7.1.2

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

**Comment by bambamal** [comment]
I agree, having only the 'show on top' option for sorting is a big annoyance and very unintuitive. A normal manual sorting behaviour would be much more practical.
